docs/en/sql-reference/statements/create/table.md

```diff
@@ -259,3 +259,68 @@ CREATE TEMPORARY TABLE [IF NOT EXISTS] table_name
 In most cases, temporary tables are not created manually, but when using external data for a query, or for distributed `(GLOBAL) IN`. For more information, see the appropriate sections
 
 It’s possible to use tables with [ENGINE = Memory](../../../engines/table-engines/special/memory.md) instead of temporary tables.
+
+## REPLACE TABLE {#replace-table-query}
+
+`REPLACE` query allows you to add, drop, and rename data in the tables.
+
+!!!note "Note"
+    This query is supported only for [Atomic](../../../engines/database_engines/atomic.md) database engine.
+
+If you need to reject(delete) some data from a table, you can create a new table and fill it with a `SELECT` statement that doesn't retrieve unwanted data, then rename it to old table name:
+
+```sql
+CREATE TABLE myNewTable AS myOldTable;
+```
+
+Then load data with:
+
+```sql
+INSERT INTO myNewTable SELECT * FROM myOldTable WHERE CounterID <12345;
+```
+
+Drop old table and rename:
+
+```sql
+DROP TABLE myOldTable;
+RENAME TABLE myNewTable TO myOldTable;
+```
+
+Instead of above, you can use the following:
+
+```sql
+REPLACE TABLE myOldTable SELECT * FROM myOldTable WHERE CounterID <12345;
+```
+
+**Examples:**
+
+`REPLACE` for a non-existent table will cause an error.
+
+```sql
+CREATE DATABASE base ENGINE = Atomic;
+REPLACE TABLE base.t1 (n UInt64, s String) ENGINE = MergeTree ORDER BY n; -- { serverError 60 }
+```
+
```

Comment:
Example is not necessary here, typically we don't give examples with errors in documentation.